src/handlers.ts

```diff
@@ -0,0 +1,49 @@
+import { Request, Response } from "express"
+import axios from "axios"
+import { globalConfig } from "./config"
+
+export async function getRandomUnsplashImageUrl(width: number, height: number): Promise<string> {
+    // localhost:3000/?width=200&height=300
+    const randomImageUrl = 
+        `${globalConfig.unsplash.apiBaseUrl}/photos/random?client_id=${globalConfig.unsplash.apiAccessKey}`
+    const { data } = await axios.get(randomImageUrl)
+    let url = data.urls.full
+
+    const imageUrl = checkWidthAndHeight(width,height,url)
+    return imageUrl
+}
+
+export async function getRandomImage(req: Request, res: Response) {
+    const {width,height} =req.query
+    const randomImageUrl = await getRandomUnsplashImageUrl(Number(width), Number(height))
+    res.redirect(randomImageUrl)
+}
+
+export async function getRandomSearchImageUrl(keyword:string, color:string, width:number, height:number): Promise<string>{
+    // localhost:3000/search/:keyword
+    let randomSearchImageUrl = 
+        `${globalConfig.unsplash.apiBaseUrl}/search/photos?query=${keyword}&client_id=${globalConfig.unsplash.apiAccessKey}`
+    if(color){
+        randomSearchImageUrl = 
+        `${globalConfig.unsplash.apiBaseUrl}/search/photos?query=${keyword}&color=${color}&client_id=${globalConfig.unsplash.apiAccessKey}`
+    }
```

Comment:
이게 겹치는게 있어서
```suggestion
    let randomSearchImageUrl = 
        `${globalConfig.unsplash.apiBaseUrl}/search/photos?query=${keyword}&client_id=${globalConfig.unsplash.apiAccessKey}`
    if(color){
        randomSearchImageUrl = randomSearchImageUrl + `&color=${color}`
    }
```

이런식으로 해도 될 것 같아!